deploy/ansible/roles/iroha-cluster-deploy-node/tasks/ubuntu.yml

```diff
@@ -0,0 +1,69 @@
+---
+  - setup:
+  - name: stop and remove all docker-compose containers before operations
```

Comment:
```
TASK [iroha-cluster-deploy-node : stop and remove all docker-compose containers before operations] ***
fatal: [iroha-bench2]: FAILED! => {"changed": true, "cmd": ["docker-compose", "down"], "delta": "0:00:01.077277", "end": "2018-03-23 13:28:40.066144", "msg": "non-zero return code", "rc": 1, "start": "2018-03-23 13:28:38.988867", "stderr": "\n        Can't find a suitable configuration file in this directory or any\n        parent. Are you in the right directory?\n\n        Supported filenames: docker-compose.yml, docker-compose.yaml\n        ", "stderr_lines": ["", "        Can't find a suitable configuration file in this directory or any", "        parent. Are you in the right directory?", "", "        Supported filenames: docker-compose.yml, docker-compose.yaml", "        "], "stdout": "", "stdout_lines": []}
...ignoring
```

Now, this output seems to be acceptable but still looks like something bad happens if there are no docker containers :) I would be happy if there are no docker containers — to see an absence of error messages in the output

Reply:
@neewy this is the error handling in Ansible. The error might occur during the task of pruning previously launched `iroha` and `postgres` docker containers

Reply:
AFAIK, that is the most suitable exception handling procedure in Ansible for this particular task.